Pipfile

```diff
@@ -25,6 +25,7 @@ pycryptodome = "*"
 requestsdefaulter = "*"
 google-cloud-storage = "*"
 google-cloud-pubsub = "*"
+urllib3 = "*"
```

Comment:
As with secure message we shouldn't be importing urllib3 but updating the package that manages the dependency

Reply:
Fixed.